packages/ai/src/ui/process-ui-message-stream.ts

```diff
@@ -59,7 +59,7 @@ export function createStreamingUIMessageState<
     MESSAGE_METADATA,
     InferUIDataParts<UI_DATA_PART_SCHEMAS>
   > = isContinuation
-    ? structuredClone(lastMessage)
+    ? lastMessage
```

Comment:
Removed because this was breaking the Svelte impl.

Quote from @elliott-with-the-longest-name-on-github:
"The problem is that `structuredClone` doesn't work on Svelte state (because it's a Proxy, which isn't cloneable)"